apps/test/unit/templates/instructions/TeacherFeedbackTest.jsx

```diff
@@ -0,0 +1,407 @@
+
+      // Student Has Feedback
+      wrapper.setState({
+        comment: 'Good work!',
+        performance: 'meets',
+        studentId: 1,
+        latestFeedback: [
+          {
+            comment: 'Good work!',
+            created_at: '2019-03-26T19:56:53.000Z',
+            id: 5,
+            level_id: 123,
+            performance: 'meets',
+            student_id: 1,
+            teacher_name: 'Tim The Teacher'
+          }
+        ],
+        submitting: false
+      });
+
+      // Key Concept
+      expect(wrapper).to.contain('Key Concept');
+      expect(wrapper).to.contain('This is the Key Concept');
+
+      // Rubric
+      expect(wrapper.find('RubricField')).to.have.lengthOf(4);
+      const confirmExceedsRatioButton = wrapper.find('RubricField').at(1);
+      expect(confirmExceedsRatioButton.props().disabledMode).to.equal(true);
+      expect(confirmExceedsRatioButton.props().rubricLevel).to.equal('meets');
+      expect(confirmExceedsRatioButton.props().rubricValue).to.equal(
+        'met expectations'
+      );
+      expect(confirmExceedsRatioButton.props().currentlyChecked).to.equal(true);
+
+      // Comment
+      const confirmCommentArea = wrapper.find('CommentArea').at(0);
+      expect(confirmCommentArea.props().disabledMode).to.equal(true);
+      expect(confirmCommentArea.props().studentHasFeedback).to.equal(true);
+      expect(confirmCommentArea.props().comment).to.equal('Good work!');
+
+      // Submit Feedback
+      expect(wrapper.find('Button')).to.have.lengthOf(0);
+    });
+    it('Rubric - Performance Given', () => {
+      const wrapper = shallow(<TeacherFeedback {...STUDENT_RUBRIC_PROPS} />);
+
+      // Student Has Feedback
+      wrapper.setState({
+        comment: '',
+        performance: 'meets',
+        studentId: 1,
+        latestFeedback: [
+          {
+            comment: 'Good work!',
+            created_at: '2019-03-26T19:56:53.000Z',
+            id: 5,
+            level_id: 123,
+            performance: 'meets',
+            student_id: 1,
+            teacher_name: 'Tim The Teacher'
+          }
+        ],
+        submitting: false
+      });
+
+      // Key Concept
+      expect(wrapper).to.contain('Key Concept');
+      expect(wrapper).to.contain('This is the Key Concept');
+
+      // Rubric
+      expect(wrapper.find('RubricField')).to.have.lengthOf(4);
+      const confirmExceedsRatioButton = wrapper.find('RubricField').at(1);
+      expect(confirmExceedsRatioButton.props().disabledMode).to.equal(true);
+      expect(confirmExceedsRatioButton.props().rubricLevel).to.equal('meets');
+      expect(confirmExceedsRatioButton.props().rubricValue).to.equal(
+        'met expectations'
+      );
+      expect(confirmExceedsRatioButton.props().currentlyChecked).to.equal(true);
+
+      // Comment
+      expect(wrapper.find('CommentArea')).to.have.lengthOf(0);
+
+      // Submit Feedback
+      expect(wrapper.find('Button')).to.have.lengthOf(0);
+    });
```

Comment:
I like how much detail you've put into these tests.  In a lot of ways these remind me of storybook entries, but you're actually checking that specific features are set up as expected.

If you're going to write more tests in this style, you may want to get familiar with `chai-enzyme`'s [`containMatchingElement`](https://github.com/producthunt/chai-enzyme#containmatchingelementnode).  There's some subtlety to its behavior that you have to learn but it can be a lot more compact and assert much of what you're saying here.